examples/demo_dlmbl/solution.py

```diff
@@ -1,53 +1,80 @@
 # %% [markdown]
 """
-# Image translation
----
+<div style="text-align: left;">
+    <h1>Image translation (Virtual Staining)</h1>
+    <hr>
+    <h3>Overview</h3>
+    <p>In this exercise, we will solve an image translation task to predict fluorescence images of nuclei and membrane markers from quantitative phase images of cells. In other words, we will <i>virtually stain</i> the nuclei and cell membrane visible in the phase image. We will apply a series of spatial and intensity augmentations to train robust models and evaluate their performance. Finally, we will apply the inverse process of predicting a phase image from a fluorescence membrane label.</p>
+
+    <div style="text-align: center;">
+        <br><br>
+        <figure>
+            <a href="https://github-production-user-asset-6210df.s3.amazonaws.com/67518483/343891935-d53a81eb-eb37-44f3-b522-8bd7bddc7755.mp4?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAVCODYLSA53PQK4ZA%2F20240705%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240705T004603Z&X-Amz-Expires=300&X-Amz-Signature=8fa0e0f15e8925c8c1bbafa6d5531c4c3a725a677df82ba780097747cbdc993d&X-Amz-SignedHeaders=host&actor_id=10470962&key_id=0&repo_id=647381053" target="_blank">
```

Comment:
Looks like you want to point the reader to the movie. That is a very good idea. The link didn't work for me.

Reply:
Thanks for checking this. We changed this to point to the same link as the README.md rather than from when you click the video.